### ISSUE TITLE
One User LDAP for Multiple Domains

### ISSUE THREAD

**Issue by xanpri** [Ylianst/MeshCentral#4637]
Currently today in my configuration MeshCentral version 1.0.0 is creating one user per domain (example 01). Would it be possible to release a user to multiple domains without the need to create a user per domain (example 02)?

Example 01:
```
USER	MAIL			DOMAIN		PERMISSION
user01	user01@domain.com	domain01	Administrator
user01	user01@domain.com	domain02	Administrator
user01	user01@domain.com	domain03	Administrator
user02	user02@domain.com	domain01	User
user02	user02@domain.com	domain02	User
user02	user02@domain.com	domain03	User
```

Example 02?
```
USER	MAIL			DOMAIN				PERMISSION
user01	user01@domain.com	domain01/domain02/domain03	Administrator
user02	user02@domain.com	domain01/domain02/domain03	User
```

**Comment by TerAnYu** [comment]
#4571 - duplicate

**Comment by Ylianst** [comment]
In this request, is the word "domain" indicate a MeshCentral domain. Like, you have multiple domains in the MeshCentral config.json? If so, are you settings up the same LDAP configuration in each MeshCentral domain? I am not sure why you would use multiple MeshCentral domains in this case. Like I indicated in #4571, I am not sure domains what should be used in this case. What is the usage model? Why have multiple domains with the same users? - Thanks

**Comment by xanpri** [comment]
Good afternoon,
I work in a government agency and I used it by group of devices and recently I started using domains, for example: meshcentral.com/dominio.
I started to separate by domains because I have a team that supports the entire government and others just for some government bodies. And when it passes 5000 devices the websocket is very slow to load and even to search the device.
When I divided by meshcentral.com/dominio this slowness stopped happening.
Today I have 14,000 devices and the trend is to increase that number.
